doc/tutorial.rst

```diff
@@ -192,9 +187,9 @@ do with the remaining warnings.
 
 If we add a docstring to describe what the code is meant to do that will help.
 There are 5 ``invalid-name`` messages that we will get to later.  Lastly, I
-violated the convention of using spaces around an operator such as ``=`` so I'll
-fix that too. To sum up, I'll add a docstring to line 2, and put spaces around
-the ``=`` sign on line 16.
+put an unnecessary semicolon at the end of the import line so I'll
```

Comment:
I do not understand why we should change this?
If you let the = sign without spaces around then the message will be emitted by `pylint`.
Why would you prefer illustrating the unnecessary semicolon from the missing spaces around operator?

Reply:
Pylint 1.6.0 does not check for spaces around the = sign.

Reply:
Sorry for my inappropriate remark.